crates/solidity/outputs/cargo/slang_solidity/src/generated/mod.rs

```diff
@@ -15,6 +15,9 @@ pub mod parse_output;
 #[cfg(feature = "slang_napi_interfaces")]
 pub mod napi_interface;
 
+#[cfg(feature = "cli")]
+pub mod cli;
```

Comment:
Feature `cli` is turned on by default (important for `cargo install`). So this module would be accessible to all users. And IIUC, we cannot make it `pub(crate)` because it won't be accessible to `main.rs` (which is treated as a different crate).

I'm not sure what is the solution here 😞 I will keep thinking about it, but I think this is OK for now to unblock you.

Reply:
Ok, I think I found a good compromise: I put all graph builder related code under a private feature flag `__graph_builder`, so the neither new API nor the `metaslang_graph_builder` dependency are exposed by default.

Reply:
Great!